### DISCUSSION TITLE
Provide Oracle 23c Free option in addition to existing Oracle XE option

### DISCUSSION THREAD

**Discussion by markxnelson** [testcontainers/testcontainers-java#7682] [Ideas]
Hi,
I am interested in contributing, but I wanted to check with you first.  Currently there is a module/container type for Oracle XE - see https://testcontainers.com/modules/oracle-xe/ 
Oracle 23c has a free developer version - see https://www.oracle.com/database/free/
This has more features and is more up to date than Oracle XE.
I'd like to contribute a PR to allow a choice between these two options.  I'd like to get a feeling for whether you'd be interested in that, before I start work, as suggested in the contribution guidelines. 
Thanks,
Mark Nelson (Oracle) - I work on Oracle Backend for Spring Boot **http://bit.ly/oraclespringboot**

**Comment by eddumelendez** [comment]
Hi, thanks for the request. See https://github.com/testcontainers/testcontainers-java/pull/7624

**Reply by markxnelson** [reply]
thanks!

**Reply by eddumelendez** [reply]
oracle-free module is available in Testcontainers 1.19.2. See https://java.testcontainers.org/modules/databases/oraclefree/
